src/gradescopeapi/_classes/_course_helpers.py

```diff
@@ -106,3 +113,64 @@ def scrape_courses_info(
             course = course.find_next_sibling("a")
 
     return all_courses, is_instructor
+
+
+def get_course_members(soup: BeautifulSoup) -> list[Member]:
+    """
+    Scrape all course members from the membership page of a Gradescope course.
+
+    Args:
+        soup (BeautifulSoup): BeautifulSoup object with parsed HTML.
+
+    Returns:
+        List: A list of Member objects containing all course members' info.
+
+        For example:
+        [
+            Member(...),
+            Member(...)
+        ]
+    """
+
+    member_list = []
+
+    # maps role id to role name
+    id_to_role = {"0": "Student", "1": "Instructor", "2": "TA", "3": "Reader"}
+
+    # find all rows with class rosterRow (each row is a member)
+    roster_rows = soup.find_all("tr", class_="rosterRow")
+
+    for row in roster_rows:
+        # get all table data for each row
+        cells = row.find_all("td")
+
+        # get data from first cell
+        cell = cells[0]
+
+        data_button = cell.find("button", class_="rosterCell--editIcon")
+
+        # fetch full name from data-cm attribute in button
+        data_cm = data_button.get("data-cm")
+        json_data_cm = json.loads(data_cm)  # convert to json
+        full_name = json_data_cm.get("full_name")
+
+        # fetch email, id, and role from data attributes in button
+        email = data_button.get("data-email")
+        id = data_button.get("data-id")
+        role = id_to_role[data_button.get("data-role")]
```

Comment:
```suggestion
        role = id_to_role[data_button.get("data-role")]
        sections = id_to_role[data_button.get("data-sections")]
```
Can capture which section a user is in too